contracts/registry/src/commitments.rs

```
@@ -109,6 +111,10 @@ impl VoteTally {
             _ => {}
         }
     }
+=======
```

Clippy (1.97.1)

[error] non-item in item list

(error)